tests/test_iter.py

```diff
@@ -47,8 +55,7 @@ def test_trie_next_prev_using_fixtures(fixture_name, fixture):
 
 
 @given(random_trie_strategy())
-@settings(max_examples=200)
```

Comment:
Just curious, `max_examples` was dropped because iterating became faster and it isn't needed anymore to limit the examples?

Reply:
It was dropped because I remembered that 200 was the default :see_no_evil: